packit/cli/utils.py

```diff
@@ -0,0 +1,34 @@
+import functools
+import logging
+import sys
+
+from packit.exceptions import PackitException
+
+logger = logging.getLogger(__name__)
+
+
+def cover_packit_exception(_func=None, *, exit_code=1):
+    """
+    Decorator for executing the function in the try-except block.
+
+    The PackitException is caught and
+    - raised in debug
+    - sys.exit(exit_code), otherwise
+
+    """
+
+    def decorator_cover(func):
+        @functools.wraps(func)
+        def covered_func(*args, **kwargs):
+            try:
+                func(*args, **kwargs)
+            except PackitException as exc:
+                logger.exception(exc)
+                sys.exit(exit_code)
```

Comment:
honestly? `if debug: raise` would be really nice here; we can change it in future

Reply:
OK, I'll revert that change..;-)

Reply:
`raise` makes sense if there's a higher level handler that would catch it - is there any?

If there's no higher level try-except then I'd expect to see:
```python
  except PackitException as exc:
    logger.exception(exc) if debug else logger.error(exc)
    sys.exit(exit_code)
```

which logs whole traceback in case of `debug` and only the error message in other cases

Reply:
@jpopelka Is the actual state ok for you?